core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/rds/RdsConfigV4Endpoint.java

```diff
@@ -27,68 +27,68 @@
             nickname = "createRdsConfigInWorkspace")
-    RDSConfigResponse createInWorkspace(@PathParam("workspaceId") Long workspaceId, @Valid RDSConfigRequest request);
+    RDSConfigResponse create(@PathParam("workspaceId") Long workspaceId, @Valid RDSConfigRequest request);
 
     @GET
     @Path("{name}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.GET_BY_NAME_IN_WORKSPACE, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "getRdsConfigInWorkspace")
-    RDSConfigResponse getByNameInWorkspace(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
+    RDSConfigResponse get(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
 
     @DELETE
     @Path("{name}")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.DELETE_BY_NAME_IN_WORKSPACE, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "deleteRdsConfigInWorkspace")
-    RDSConfigResponse deleteInWorkspace(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
+    RDSConfigResponse delete(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
 
     @GET
     @Path("{name}/request")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.GET_REQUEST_IN_WORKSPACE, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "getRdsRequestFromNameInWorkspace")
-    RDSConfigRequest getRequestFromName(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
+    RDSConfigRequest getRequest(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name);
 
     @POST
     @Path("testconnect")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.POST_CONNECTION_TEST, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "testRdsConnectionInWorkspace")
-    RdsTestResult testRdsConnection(@PathParam("workspaceId") Long workspaceId, @Valid RDSTestRequest rdsTestRequest);
+    RdsTestResponse testConnection(@PathParam("workspaceId") Long workspaceId, @Valid RDSTestRequest rdsTestRequest);
 
     @PUT
     @Path("{name}/attach")
     @Produces(MediaType.APPLICATION_JSON)
     @Consumes(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RdsConfigOpDescription.ATTACH_TO_ENVIRONMENTS, produces = ContentType.JSON, notes = Notes.RDSCONFIG_NOTES,
             nickname = "attachRdsResourceToEnvironments")
-    RDSConfigResponse attachToEnvironments(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name, @NotEmpty Set<String> environmentNames);
+    RDSConfigResponse attach(@PathParam("workspaceId") Long workspaceId, @PathParam("name") String name, @NotEmpty Set<String> environmentNames);
 
     @PUT
```

Comment:
consider the http method, is detach DELETE?

Reply:
as we agreed, attach/detach will be PUT